license/src/com/untangle/app/license/LicenseManagerImpl.java

```diff
@@ -1010,12 +1011,35 @@ private void _syncLicensesWithServer()
                     _saveSettings(this.settings);
                 }
             }
+            _cleanLicenseFiles();
             _runAppManagerSync();
         }
 
         logger.info("Reloading licenses... done" );
     }
 
+    /**
+     * Pruning old license files, keeping last 5 license files.
+     */
+    private void _cleanLicenseFiles() {
+        logger.info ("Cleaning old license files...");
+        File licenseDirectory = new File(System.getProperty("uvm.conf.dir") + "/licenses/");
+        try {
+            if (licenseDirectory.isDirectory()) {
+                File[] files = licenseDirectory.listFiles();
+                if (files != null && files.length > 5) {
```

Comment:
Let's  make 5 be a constant definition in the class.

Reply:
@cblaise-untangle Added constant LICENSE_FILES_COUNT